nginx/domains/firmware.ffmuc.net.conf

```diff
@@ -24,6 +33,11 @@ server {
     location ~ ^/(stable|testing|experimental)/ {
         # for accessing it e.g. via ffmuc.net/firmware
         add_header Access-Control-Allow-Origin "*";
+        if($autoupdater) {
+            if ($firmwareVersion < "2024.4.2") {
```

Comment:
We probably also want to filter for `> v2022.5` 😉

Reply:
Actually, we probably want to redirect ALL routers that do not have an x-firmware-tag header as well. There might still be some legacy devices out there that don't use the x-firmware-header version tag yet.

Reply:
and maybe `$firmwareVersion != "unknown"`